index.js

```diff
@@ -25,7 +25,8 @@ if (!Element.prototype.matches) {
 AFRAME.registerComponent('teleport-controls', {
   schema: {
     type: {default: 'parabolic', oneOf: ['parabolic', 'line']},
-    button: {default: 'trackpad', oneOf: ['trackpad', 'trigger', 'grip', 'menu']},
+    aimEvent: {default: 'aim'},
+    teleportEvent: {default: 'teleport'},
```

Comment:
You have an infinite loop since the hand emits teleport but also listens to it.

Reply:
Nice catch! ;) I believe using `teleported` when the event already happened makes more sense.
Do you have any proposals for the aim and teleport events? I'm not sure if including something related to the component itself to clarify the app code: `teleportAim` and `teleportJump` maybe?

Reply:
teleportstart, teleportend